develop/CICD/GitHub/GitHubReusableWorkflows/GitHub_Reusable_Workflows.md

```diff
@@ -19,11 +19,18 @@ These reusable workflows are public and can be accessed by any organization. The
 
 You can find all available reusable workflows in the [_ReusableWorkflows](https://github.com/SkylineCommunications/_ReusableWorkflows/tree/main/.github/workflows) repository.
 
-For more details about what each workflow does, you can check one of the below links:
+For more details about what each workflow does, you can check one of the links below:
 
 - [Automation Master Workflow](xref:github_reusable_workflows_automation_master_workflow)
 - [NuGet Solution Master Workflow](xref:github_reusable_workflows_nuget_solution_master_workflow)
 - [SRM Function Master Workflow](xref:github_reusable_workflows_srm_function_master_workflow)
+- [Connector Master Workflow](xref:github_reusable_workflows_connector_master_workflow)
+
+## GitHub UI to Catalog Details
+
+This workflow utilizes a tool that auto-generates an `auto-generated-catalog.yml` file, which can extend an existing `catalog.yml` (or `manifest.yml`) file by adding metadata and registration details for a catalog item. To function, the GitHub repository must infer the catalog item type using either naming conventions or GitHub topics.
+
+For more information, see the [GitHubToCatalogYaml README](https://github.com/SkylineCommunications/Skyline.DataMiner.CICD.Tools.GitHubToCatalogYaml#readme-body-tab), specifically the section on [Inferring Catalog Item Type](https://github.com/SkylineCommunications/Skyline.DataMiner.CICD.Tools.GitHubToCatalogYaml?tab=readme-ov-file#inferring-catalog-item-type).
```

Comment:
This section seems to be repeated exactly in GitHub_Reusable_Workflows_Update_Catalog_Details.md. What is more, it's not really clear what it's doing here. If it's about another reusable workflow, why don't we just link to the page related to that workflow like we do for the others above?

Reply:
That is a new stage that's actually in almost all the reusable workflows, you'll see it duplicated in Automation, Connector, SRM and Update_Catalog_Details. I think I added it there as well for clarity, but it could be removed indeed.